contrib/pltcl/Trunk.toml

```
@@ -0,0 +1,22 @@
+[extension]
+name = "pltcl"
+version = "1.0.0"
+repository = "https://github.com/postgres/postgres/tree/master/src/pl/tcl"
+description = "The PL/Tcl procedural language for PostgreSQL."
+homepage = "https://www.postgresql.org"
+documentation = "https://www.postgresql.org/docs/current/pltcl.html"
+categories = ["procedural_languages"]
+
+[build]
+postgres_version = "15"
+platform = "linux/amd64"
+dockerfile = "Dockerfile"
+install_command = """
+    make -C postgres/src/pl/tcl install
+    set -x
+    mv /usr/local/pgsql/share/extension/pltcl.* /usr/share/postgresql/15/extension
+    mv /usr/local/pgsql/share/extension/pltcl--* /usr/share/postgresql/15/extension
```

Could this be one command? Something like:
```suggestion
    mv /usr/local/pgsql/share/extension/pltcl* /usr/share/postgresql/15/extension
```

---

For share dir its important, as we need to capture both pltcl.control vs pltclu.control, and also the pltcl--sql vs pltclu--sql.

Libdir can get away with pltcl* as it's only the pltcl.so file both for the trusted and untrusted variant.

---

Got it. I didn't realize we were avoiding the `u` files here.